### DISCUSSION TITLE
need help with materials (C++)

### DISCUSSION THREAD

**Discussion by bigidea-kz** [google/filament#7772] [General]
Hello, everyone. I’m learning Filament (C++), and I’m having trouble loading or creating materials in my application. I’ve tried compiling and loading them into the app, but I keep encountering errors. I’ve also attempted to create materials within the app, but that hasn’t been successful either. In all the examples I’ve seen, materials are compiled alongside the application. Can anyone share an example? 😊

**Comment by romainguy** [comment]
There are several examples in the source tree. What errors did you run into specifically?
